### ISSUE TITLE
jo: Cannot open error. Issues handling values with semi-colons

### ISSUE THREAD

**Issue by bitsofinfo** [jpmens/jo#123]
```
bitsofinfo@laptop1 tmp $ bash --version
bash --version
GNU bash, version 4.4.19(1)-release (x86_64-apple-darwin17.3.0)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

bitsofinfo@laptop1 tmp $ jo -v
jo -v
jo 1.3

bitsofinfo@laptop1 tmp $ jo -p x=':x: hi'
jo -p x=':x: hi'
x: hi: No such file or directory
jo: Cannot open x: hi for reading

bitsofinfo@laptop1 tmp $ jo -p x='\:x\: hi'
jo -p x='\:x\: hi'
{
   "x": ":x\\: hi"
}

bitsofinfo@laptop1 tmp $ jo -p x=":x:"
jo -p x=":x:"
x:: No such file or directory
jo: Cannot open x: for reading

bitsofinfo@laptop1 tmp $ jo -p x="\:x\:"
jo -p x="\:x\:"
{
   "x": ":x\\:"
}

bitsofinfo@laptop1 tmp $ jo -p x="\:x:"
jo -p x="\:x:"
{
   "x": ":x:"
}
```

**Comment by gromgit** [comment]
That's expected behavior. From the man page:

> Read element values from files: a value which starts with `@` is read in plain whereas if it begins with a `%` it will be base64-encoded and **if it starts with `:` the contents are interpreted as JSON**: [...]
>
>     $ jo nested=:nested.json
>     {"nested":{"field1":123,"field2":"abc"}}

Escaping the initial `:` with a backslash is sufficient to prevent this interpretation, as you discovered.

That said, I think all this should really be documented up top, since literals beginning with those characters are perfectly legal JSON strings, and therefore likely to trip up unsuspecting users. I'll see if I can come up with the appropriate wording.

**Comment by gromgit** [comment]
Documentation has been clarified. The changes have been merged into HEAD, but you can apply PR #125 to your own local source if needed.
